[2024-12-10T07:41:34.110+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-10T07:41:34.243+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_pipeline.check_api_online scheduled__2024-12-09T00:00:00+00:00 [queued]>
[2024-12-10T07:41:34.322+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_pipeline.check_api_online scheduled__2024-12-09T00:00:00+00:00 [queued]>
[2024-12-10T07:41:34.323+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 2
[2024-12-10T07:41:34.456+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): check_api_online> on 2024-12-09 00:00:00+00:00
[2024-12-10T07:41:34.533+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2718) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-10T07:41:34.546+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_data_pipeline', 'check_api_online', 'scheduled__2024-12-09T00:00:00+00:00', '--job-id', '100', '--raw', '--subdir', 'DAGS_FOLDER/portland_weather_data_pipeline.py', '--cfg-path', '/tmp/tmpthcoo6jl']
[2024-12-10T07:41:34.653+0000] {standard_task_runner.py:72} INFO - Started process 2720 to run task
[2024-12-10T07:41:34.734+0000] {standard_task_runner.py:105} INFO - Job 100: Subtask check_api_online
[2024-12-10T07:41:35.092+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_data_pipeline.check_api_online scheduled__2024-12-09T00:00:00+00:00 [running]> on host 4ccd8b9315ad
[2024-12-10T07:41:36.546+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data_pipeline' AIRFLOW_CTX_TASK_ID='check_api_online' AIRFLOW_CTX_EXECUTION_DATE='2024-12-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-09T00:00:00+00:00'
[2024-12-10T07:41:36.551+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-10T07:41:36.551+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-10T07:41:36.552+0000] {logging_mixin.py:190} INFO - Current task name:check_api_online state:running start_date:2024-12-10 07:41:34.298155+00:00
[2024-12-10T07:41:36.563+0000] {logging_mixin.py:190} INFO - Dag name:weather_data_pipeline and current dag run status:running
[2024-12-10T07:41:36.571+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-10T07:41:38.838+0000] {logging_mixin.py:190} INFO - API is online.
[2024-12-10T07:41:38.874+0000] {python.py:240} INFO - Done. Returned value was: True
[2024-12-10T07:41:39.104+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-10T07:41:39.251+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=check_api_online, run_id=scheduled__2024-12-09T00:00:00+00:00, execution_date=20241209T000000, start_date=20241210T074134, end_date=20241210T074139
[2024-12-10T07:41:39.413+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-12-10T07:41:39.414+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-12-10T07:41:39.415+0000] {logging_mixin.py:190} INFO - Dag name:weather_data_pipeline queued_at:2024-12-10 00:23:42.697892+00:00
[2024-12-10T07:41:39.415+0000] {logging_mixin.py:190} INFO - Task hostname:4ccd8b9315ad operator:PythonOperator
[2024-12-10T07:41:39.610+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-10T07:41:39.776+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2024-12-10T07:41:40.135+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-10T07:41:40.336+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
